docs/index.md

```diff
@@ -10,6 +10,26 @@ Komponenterna är testade både manuellt och med hjälp av automatiserade verkty
 
 {@link guide Läs mer om designsystemet}
 
+## Test
```

Comment:
Detta ska bort innan merge, bara så vi snabbt kan se resultatet under granskning

Reply:
Tydligen passas inte options in korrekt så det här verkar inte lira som det ska